File: example_script.py
import subprocess

def run_command(command):
    result = subprocess.run(command, shell=True, capture_output=True, text=True)
    if result.returncode != 0:
        print(f'Error: {result.stderr}')
    else:
        print(result.stdout)

if __name__ == '__main__':
    commands = [
        'echo "This is the CI tool!"',
        'ls -l',
    ]
    for command in commands:
        run_command(command)

File: example_script.js
const { exec } = require('child_process');

const runCommand = (command) => {
    exec(command, (error, stdout, stderr) => {
        if (error) {
            console.error(`Error: ${stderr}`);
            return;
        }
        console.log(stdout);
    });
};

const commands = [
    'echo "This is the CI tool!"',
    'ls -l',
];

commands.forEach(runCommand);
